### DISCUSSION TITLE
RTLSDR Blog V3 using Direct Sampling (Q Branch) and Modulation (USB) for HF Airband frequencies

### DISCUSSION THREAD

**Discussion by n3eds** [rtl-airband/RTLSDR-Airband#437] [Q&A]
I have all working fine with multiple dongles, am, nfm modulations fine.  Wondering if I could also configure for HF 8.846mHz and others (ARINC) frequencies.  I am using a true RTLSDR Blog V3 dongle so direct sampling is possible (Q Branch) below 25mHz and also would need usb modulation.  Is this possible?  I can't find an example of those in any config files to go by.  Using an R-PI 3B+ compiled for generic Linux and nfm enabled if any of that matters.  Thanks!

Jeff P.
N3EDS

**Comment by charlie-foxtrot** [comment]
No, not supported today.

Reference to SSB feature request - #403
